### PR TITLE
feat: add arg parsing

### PR DESCRIPTION
**Pull request by luan-dev**
# Summary

Add [`click`](https://click.palletsprojects.com/en/8.1.x/)

### CODE REVIEW

File: pdf_to_ppt/setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  name="pdf_to_ppt",
+  version="0.1.0",
+  description="Converts PDF to PowerPoint",
+  url="https://github.com/luan-dev/tools",
+  author="Luan <github.com/luan-dev>",
+  license="GPLv3",
+  packages=["converter"],
+  include_package_data=True,
+  install_requires=[
+        "PyQt5",
```
Comment (dionboles-asym) [inline pdf_to_ppt/setup.py]: add all of you install requirements here so if you do the pip install -e . setup.py will have all of your dependents

File: pdf_to_ppt/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+fbs==1.1.2
+future==0.18.2
 lxml==4.8.0
+macholib==1.16
 pdf2image==1.16.0
+pefile==2021.9.3
 Pillow==9.0.1
 python-pptx==0.6.21
-XlsxWriter==3.0.2
+XlsxWriter==3.0.3
```
Comment (dionboles-asym) [inline pdf_to_ppt/requirements.txt:11]: check out pip-tools it can keep track of your library’s

**Review by dionboles-asym — APPROVED** [review]

File: pdf_to_ppt/setup.py (hunk 1 above)
Comment (luan-dev) [reply pdf_to_ppt/setup.py]: this was an artifact of my pyqt5 attempt

File: pdf_to_ppt/requirements.txt (hunk 2 above)
Comment (luan-dev) [reply pdf_to_ppt/requirements.txt:11]: i'll add that in a future PR

**Pull request merged**